operator_ui/src/actions.ts

```diff
@@ -328,7 +328,7 @@ export const fetchRecentlyCreatedJobs = (size: number) =>
   )
 
 export const fetchJob = (id: string) =>
-  request('JOB', api.getJobSpec, (json: object) => normalize(json), id)
+  request('JOB', api.getJobSpec, (json: object) => normalize(json, { camelizeKeys: false }), id)
```

Comment:
@rupurt @NavyAdmiral Should this setting be on all actions? Would creating a `normalize` wrapper that defaults to this be overkill?

Reply:
I don't think that would hurt. Golang keys are already camelized where necessary, and doing this will only bring consistency.

Reply:
I don't feel like this should be the default. In JS/TS the convention is camel case. 

I'm looking at the response payload and it seems like the keys are a mix of camel case + title case? Which keys are the problem and need to maintain their casing? 

![Screen Shot 2019-07-29 at 1 31 48 PM](https://user-images.githubusercontent.com/680789/62080412-74764780-b205-11e9-95a1-0163600ec4e9.png)

Reply:
Looked at the story. It's the header keys 😄

Reply:
Taking a look after reading the story. Adding `camelCase: false` for this endpoint makes sense I feel like we should also update the response payload from the server so that it returns `camelCase` for all keys.

Reply:
*Adding clarification

> I feel like we should also update the response payload from the server so that it returns camelCase for all keys.

I feel like we should also update the response payload from the server so that it returns camelCase for all _non-user input_ keys. e.g. `ID` -> `id`